pkg/internal/pipe/config.go

```diff
@@ -46,18 +46,22 @@ var defaultConfig = Config{
 			Path: "/internal/metrics",
 		},
 	},
+	Kubernetes: transform.KubernetesDecorator{
+		Enable:               transform.KubeDisabled,
```

Comment:
We can't because actually the `Enabled()` function in K8s.go uses this `Enable` property to decide whether the node is enabled, disabled, or needs to be autodetected.

The current configuration, from the user perspective would be as defaulting:
```yaml
kubernetes:
    enabled: false
```
I will rename the constants to avoid  that confusion.

Reply:
Renamed constants to: `EnableTrue`, `EnableFalse`, `EnableAutodetect` and added an `EnableDefault`